src/js/pagination/paging-collection.js

```diff
@@ -83,7 +83,15 @@
              * backbone.paginator.
              */
             parseState: function (response, queryParams, state, options) {
-                return {totalRecords: response[0].count, totalPages: response[0].num_pages};
+                state.totalRecords = response[0].count;
```

Comment:
This shouldn't mutate `state`. See: https://backbone-paginator.github.io/backbone.paginator/api/#parseState - "This method MUST return a new state object instead of directly modifying the PageableCollection#state object. The behavior of directly modifying PageableCollection#state is undefined."

Please follow the previous pattern; construct and return a new object.